src/drivers/imx/ipc.c

```diff
@@ -23,6 +23,7 @@
 #include <ipc/topology.h>
```

Comment:
@tlauda do we have a real usecase where checking IPC bits is not enough?

Reply:
@dbaluta Yes, during specific power flows on cAVS DSP.